tools/EventClients/Clients/PS3 BD Remote/ps3_remote.py

```diff
@@ -95,7 +95,7 @@ def get_remote_address(remote, target_name = "BD Remote Control"):
                 remote.connect((target_address,19))
                 target_connected = True
                 print "Remote Paired.\a"
-                xbmc.send_notification("Pairing Successfull",
```

Comment:
I thaught the task is to fix spelling.. :)
Anyways, fine for me to leave it out, it´s still an improvement.